lib/components/viewers/nearby/stop-card-header.tsx

```diff
@@ -1,6 +1,6 @@
 import { connect } from 'react-redux'
 import { FormattedMessage, useIntl } from 'react-intl'
-import { Search } from '@styled-icons/fa-solid/Search'
+import { MagnifyingGlassPlus } from '@styled-icons/fa-solid/MagnifyingGlassPlus'
```

Comment:
Ugh also i think i prefer the normal looking glass icon sorry, this one just feels like there's a lot going on. I can be convinced to keep it